provisioning/miniprov/hare_mp/main.py

```diff
@@ -53,8 +58,27 @@ def execute(cmd: List[str]) -> str:
     return out
 
 
-def setup_logging():
-    logging.basicConfig(level=logging.INFO, format='%(message)s')
+def create_logger_directory():
+    """Create log directory if not exsists."""
+    _logger_directory = os.path.join(LOG_DIR)
```

Comment:
Why is it needed?

Reply:
If there is no any directory is available so logically we can create for it So we can avoid future issues. Only I use one variable for assigning full path of directory.

Reply:
I changed _logger_directory to LOG_DIR and removed os.path.join()  this parameter.